### DISCUSSION TITLE
Open file file with path and line

### DISCUSSION THREAD

**Discussion by oliver-kriska** [zed-industries/zed#6551] [Feature Requests]
Hello,
I'm used to copy file path with line number at the end and open it. But when I enter for example this:
` test/platform/orders/order_service_test.exs:369` 
with cmd+p (file_finder) it doesn't work. Any idea how can I open files this way? Mostly I click in terminal on that path and it will open it automatically in other editors. That doesn't work too, so at least I can copy&paste file and line where error happen for example.

**Comment by hovsater** [comment]
It's currently not supported, but definitely will be in the future. Feel free to upvote https://github.com/zed-industries/community/issues/1184 (or https://github.com/zed-industries/community/issues/557 specifically for your use case). 🙂

**Comment by JosephTLyons** [comment]
The ability to open a file at specific line / column, in both Zed CLI and in the file finder, should be landing soon.

**Comment by JosephTLyons** [comment]
This feature is now available within Zed!
